### DISCUSSION TITLE
Dịch shallow net

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#103] [Terminology] [answered]
Một cách dịch là `mạng nông`, nhưng nghe có vẻ không hay lắm :sweat: 

cc @mlbvn/handson-ml

**Comment by quangnhat185** [comment]
`Mạng ít tầng` thì sao nhỉ?

**Comment by lkhphuc** [comment, marked as answer]
Mình thấy `mạng nông` đọc trong ngữ cảnh cũng ok mà.

**Reply by minhduc0711** [reply]
Ờm, chắc do mình nghe ít nên thấy lạ :+1:

**Reply by hnguyentt** [reply]
mình cũng thấy mạng nông okay.
